Failed to build package mesa (No module named mako.template)

```Failed to build package mesa for target i686-w64-mingw32.static!
------------------------------------------------------------
    from mako.template import Template
ImportError: No module named mako.template
scons: *** [build/windows-x86/mesa/main/format_fallback.c] Error 1
scons: building terminated because of errors.
make[1]: *** [Makefile:846: build-only-mesa_i686-w64-mingw32.static] Error 2
make[1]: Leaving directory '/opt/mxe'
```

I get this from a fresh `./build.sh` run

This upstream issue seems related: https://github.com/mxe/mxe/issues/2561

--- a/Dockerfile.mxe
+++ b/Dockerfile.mxe
@@ -10,7 +10,7 @@ RUN \
         g++ g++-multilib gettext git gperf intltool iputils-ping libc6-dev-i386 \
         libffi-dev libgdk-pixbuf2.0-dev libltdl-dev libssl-dev libtool-bin \
         libxml-parser-perl lzip make nano openssl p7zip-full patch perl \
-        pkg-config python ruby scons sed unzip wget xz-utils && \
+        pkg-config python python-mako ruby scons sed unzip wget xz-utils && \
     apt -y autoremove && \
     apt -y autoclean && \
     apt -y clean && \

